### ISSUE TITLE
[Suggestion] Replace mob tools and armor

### ISSUE THREAD

**Issue by YaibaToKen** [asiekierka/TinkeredHegemony#11]
As per the title, replacement of mob's tools and armor would be welcome :)
Perhaps define a a couple of base materials for each tool / armor part as a fallback when no adequate material is found in the TinkersRegistry.

**Comment by asiekierka** [comment]
Sounds like a good idea!

**Comment by YaibaToKen** [comment]
As a side note, is there a way to create an addon expanding what gets replaced by this mod?

**Comment by asiekierka** [comment]
Not really. It's pretty specific to TCon and TCon addons in its design at this stage.

**Comment by YaibaToKen** [comment]
Good point, I just need to check if Tinkered Hegemony is loaded and if so, apply recipe changes. (I happen to be the author to Armory Expansion so.. I already have an add-on, those are some of the planned features.) The troubling part is setting up the models and textures x.x

**Comment by YaibaToKen** [comment]
Any news on this? And is there anything I can help in regards to this?
